client-dynamodb-v2/src/main/kotlin/app/cash/backfila/client/dynamodbv2/UpdateInPlaceDynamoDbBackfill.kt

```diff
@@ -31,7 +31,7 @@ abstract class UpdateInPlaceDynamoDbBackfill<I : Any, P : Any>(
             dynamoDbTable.tableName() to itemsToSave.map {
               WriteRequest.builder().putRequest(
                 PutRequest.builder().item(
-                  dynamoDbTable.tableSchema().itemToMap(it, false),
+                  dynamoDbTable.tableSchema().itemToMap(it, true),
```

Comment:
What does the true/false toggle here?

Reply:
If someone ever requires this. Then that means they would know how to configure Dynamo to handle this properly. I couldn't figure this out from reading the docs and searching for answers.

I think this may be a forward looking feature rather than what the current state can accept.